project.clj

```diff
@@ -26,4 +28,5 @@
                         :optimizations :none
                         :pretty-print false}}]}
   :doo {:alias {:browsers [:chrome :firefox]}
-        :paths {:karma "./node_modules/karma/bin/karma"}})
+        :paths {:karma "./node_modules/karma/bin/karma"}}
+  :profiles {:dev {:dependencies [[pjstadig/humane-test-output "0.8.2"]]}})
```

Comment:
It looked like I had to activate it in `runner.cljs` like [this](https://github.com/clojure-tw/hopen/blob/605af718ba9522301856e23c018e873e3dcb2924/test/hopen/runner.cljs#L4).  I don't think it would compile if I leave out the dependency.

This appears to be the way to add dependencies that should only be included for dev or test time. [link](https://stackoverflow.com/questions/44022911/clojure-dependencies-for-tests-only)

---

I think having a default test reporter for both clj and cljs would be useful to anyone who might want to participate in working on this project.

The main problem is that the default test reporter show something like this for a failing case:
![Screen Shot 2019-04-11 at 10 01 29 AM](https://user-images.githubusercontent.com/10692/55925677-e750b800-5c40-11e9-8f8f-5ade0b696b0a.png)

It's quite difficult to figure out what's actually different here. I've been using eftest with clj so far (added via profile.clj).  It prints the error this way.

![Screen Shot 2019-04-11 at 10 03 36 AM](https://user-images.githubusercontent.com/10692/55925766-372f7f00-5c41-11e9-843a-d9411e021929.png)

That's much more readable! The last `diff` section also makes it clear what is actually different in these two pieces of data. [Kaocha](https://github.com/lambdaisland/kaocha) actually produces even better looking output (and it's faster to run for some magical reason).

Both `eftest` and `Kaocha` seem to be for clj only though. For cljs, "humane-test-output" seems to be the only one that's easy to add. It's prints errors like this:

![Screen Shot 2019-04-11 at 10 09 59 AM](https://user-images.githubusercontent.com/10692/55926143-c9845280-5c42-11e9-863c-370244234f43.png)
 
It's not colorized, but still makes the test output much easier to read and understand.

Reply:
The readme file of contains information on how to avoid changing the source code.
https://github.com/pjstadig/humane-test-output#clojure

I am using IDEA and I don't need that library. What I need is the default output.

Reply:
I already tried the instructions, but they do not work for cljs.